### ISSUE TITLE
Crashes when using small icon preview

### ISSUE THREAD

**Issue by QuestDragon** [files-community/Files#14616]
### Description

Hello, I'm a Japanese Files user.
_The text may be incorrect due to the use of translations. note that._

This is the main topic, but I would like to report that there were some parts that did not work properly when I was using it.
When accessing a folder that is about **2GB in size and has 134 items in the small icon layout, it seems to crash while loading.**

### Steps To Reproduce

1. Prepare various files in one folder. (From light files such as txt to heavy files such as bmp, mp4, font files, dll files, and zip files.)
2.Create multiple folders within the folder and prepare various similar files within them.
3. Return to folder "1" and change the layout to small icons.
4. If it loads normally, close Files and reopen it.
5. Access folder “1”.
6. If you try to preview with small icons, it will crash here.
*It may not be possible to reproduce depending on the environment. It may depend on the file and what you can preview.

### Requirements

Prevent crashes when accessing folders that are set to display in _(but may not be limited to)_ the small icon layout

### Files Version

3.1.1.0

### Windows Version

10.0.19045.3930

### Log File

[debug.log](https://github.com/files-community/Files/files/14126467/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report, can you see if this is an issue in the preview version. https://files.community/download

**Comment by QuestDragon** [comment]
Thank you for your reply.
**When I tried the Preview version, it seems that this problem does not occur.**
I guess they were in the process of fixing it...I'm sorry I didn't check carefully before posting...

We support the development!

**Comment by QuestDragon** [comment]
Sorry, I have another problem with the Preview version...
It seems like I can't open the file.
**"A device connected to the system is not functioning."**
It seems like an error message is displayed...

**Comment by QuestDragon** [comment]
By the way, you can't open it by double-clicking it, but if you select "Open with" and select Notepad, etc., it will appear.

**Comment by yaira2** [comment]
I'm happy to hear the crashing issue has been resovled. 

Regarding the issue with opening text files, does it work if you uninstall/reinstall Files?

**Comment by QuestDragon** [comment]
When uninstalling, should I also uninstall the stable version?  Or is it okay to uninstall only the Preview version in question?

**Comment by QuestDragon** [comment]
![image](https://github.com/files-community/Files/assets/126067454/1e2ffc7f-46ea-4434-bd44-c2c869e6b3ea)
**I reinstalled it and it opened!**
For reference, what I did after installing the Preview version and before I encountered the problem of not being able to open files was as follows, but is there something to do with it?
- Import settings from stable version
- Confirmed the problem that occurred when this topic started _(small icon preview crash)_
- Layout display settings for each folder _(download folder, picture folder, etc.)_

**Comment by yaira2** [comment]
I'm happy to hear it's been resolved, I'm not really sure what caused it but it's possible that it's a bug in Windows.

**Comment by QuestDragon** [comment]
That's right!  
I was wondering if importing the settings from the stable version was a bad idea, but _(when I changed the settings manually without using the import function, I didn't have the problem of not being able to open the file)._ It's an amateur's idea, so I don't really understand it...lol 
Anyway, I solved it successfully!  
Thank you for your response!  
I will continue to use the Preview version for the time being until there is a stable update.  
Good luck with your development!
